book-example/src/format/mathjax.md

```diff
@@ -13,7 +13,7 @@ mathjax-support = true
 The usual delimiters MathJax uses are not yet supported. You can't currently use `$$ ... $$` as delimiters and the `\[ ... \]` delimiters need an extra backslash to work. Hopefully this limitation will be lifted soon.
 
 ### Inline equations
-Inline equations are delimited by `\\[` and `\\]`. So for example, to render the following inline equation \\( \int x dx = \frac{x^2}{2} + C \\) you would write the following:
+Inline equations are delimited by either `\\(` and `\\)` or `\\[` and `\\]`. So for example, to render the following inline equation \\( \int x dx = \frac{x^2}{2} + C \\) you would write the following:
```

Comment:
Inline equations are parentheses only, brackets are for block equations. Your first commit was right 😉